src/modules/user-settings/lib/editor.ts

```diff
@@ -14,29 +15,50 @@ export const supportedEditorConfig: Record< SupportedEditor, SupportedEditorConf
 		label: __( 'VS Code' ),
 		url: ( path: string ) => `vscode://file/${ path }?windowId=_blank`,
 		bundleId: 'com.microsoft.VSCode',
+		winPaths: [
+			'%LOCALAPPDATA%\\Programs\\Microsoft VS Code\\code.exe',
+			'%PROGRAMFILES%\\Microsoft VS Code\\code.exe',
+			'%PROGRAMFILES(X86)%\\Microsoft VS Code\\code.exe',
+		],
```

Comment:
These paths somewhat duplicate what we were already doing in is-installed, but the path here needed to be more specific. I thought it was beyond the scope of this PR to combine the two solutions.

However, it'd be nice to use `supportedEditorConfig` as the source of truth, and update `is-installed` to work with these paths. I'll create a separate PR for this.